preload/ElmSharp.preload

```diff
@@ -0,0 +1,43 @@
+#########################################
+## AssemblyName.dll ClassName          ##
+## AssemblyName.dll ClassName.Method() ##
```

Comment:
nit: ClassName → TypeName because we have Interfaces as well.

Reply:
done